pkgs/desktops/gnome/apps/gnome-characters/default.nix

```diff
@@ -38,6 +38,7 @@ stdenv.mkDerivation rec {
     python3
     desktop-file-utils
     wrapGAppsHook4
+    gjs
```

Comment:
This should be fixed by adding `native: true` kwarg to `find_program` here:

https://gitlab.gnome.org/GNOME/gnome-characters/-/blob/b72bf4e1a7cbe8446ef2659d432b78ac8b50df83/src/meson.build#L2

Reply:
find_program already looks in PATH

ther reason its needed in buildInputs is https://gitlab.gnome.org/GNOME/gnome-characters/-/blob/b72bf4e1a7cbe8446ef2659d432b78ac8b50df83/meson.build#L47

Reply:
`buildInputs` I am fine with since `gjs` is an interpreter used for running the app. But it should not be necessary at build time, as far as I can tell.

Reply:
https://gitlab.gnome.org/GNOME/gnome-characters/-/merge_requests/82